src/themes/grayscale.js

```diff
@@ -0,0 +1,145 @@
+export default {
+  line: {
+    data: {
+      stroke: "#252525",
+      strokeWidth: 2,
+      fill: "none",
+      opacity: 1
+    },
+    labels: {
+      padding: 5,
+      fontFamily: "Poppins, Helvetica-Neue, sans-serif",
+      letterSpacing: 0.2,
+      fontSize: 10,
+      strokeWidth: 0,
+      stroke: "transparent",
+      textAnchor: "start"
+    },
+    parent: {
```

Comment:
This shouldn't be nested. `parent` just refers to the containing `svg` or `containerComponent` so it should't have `data` and `labels` styles